packages/sdk/utils/base64.ts

```diff
@@ -17,3 +17,8 @@ export const fromBase64Url = (encoded: string): Buffer => {
   encoded = encoded.replace(/\-/g, '+').replace(/_/g, '/')
   return Buffer.from(encoded, 'base64')
 }
+
+export const toBase64 = (buffer: ArrayBuffer): string => {
+  const view = new Uint8Array(buffer);
+  return btoa(String.fromCharCode(...view));
```

Comment:
can we not use `Buffer.toString` here? or should we switch to `btoa` everywhere? prefer to be consistent on b64 handling.

Reply:
Yes I tried to avoid the Buffer dependency since it does not exists natively on the browser side, what do you think?
I see it is in the `sign` method though but I can change it.

Reply:
I am not a fan of having the buffer dependency for the browser but since we support both browser and node code keeping it seems cleaner for now. I removed `btoa` usage and cleaned base64 code.

Reply:
yeah I'm not against switching over to `btoa` across the board. it's your call if you think it's not worth taking a dependency.